packages/cfd/src/Components/__tests__/cfd-account-card.spec.tsx

```diff
@@ -0,0 +1,484 @@
+            />
+        );
+        expect(screen.getByText(/most popular/i)).toBeInTheDocument();
+        expect(screen.getByText(/demo/i)).toBeInTheDocument();
+        expect(screen.getByText(/IcMt5CfdPlatform/i)).toBeInTheDocument();
+        expect(screen.getAllByText(/synthetic/i)[0]).toBeInTheDocument();
+        expect(
+            screen.getByText(/trade cfds on our synthetic indices that simulate real-world market movement./i)
+        ).toBeInTheDocument();
+        expect(screen.getByText(/20103240/i)).toBeInTheDocument();
+        expect(screen.getByText(/usd/i)).toBeInTheDocument();
+        expect(screen.getByRole('button', { name: /top up/i })).toBeInTheDocument();
+        expect(screen.getByRole('button', { name: /trade/i })).toBeInTheDocument();
+    });
+
+    it('should display the component for Demo MT5 Financial account ', () => {
+        const type = {
+            type: 'financial',
+            category: 'demo',
+            platform: 'mt5',
+        };
+        render(
+            <CFDAccountCard
+                {...props}
+                type={type}
+                descriptor={financial_descriptor}
+                title='Financial'
+                existing_accounts_data={[mt5_acc]}
+            />
+        );
+        expect(screen.getByText(/demo/i)).toBeInTheDocument();
+        expect(screen.getByText(/IcMt5CfdPlatform/i)).toBeInTheDocument();
+        expect(screen.getAllByText(/financial/i)[0]).toBeInTheDocument();
+        expect(
+            screen.getByText(
+                /trade major \(standard & micro-lots\) and minor forex, stocks & stock indices, commodities, basket indices, and crypto with high leverage./i
+            )
+        ).toBeInTheDocument();
+        expect(screen.getByText(/20103240/i)).toBeInTheDocument();
+        expect(screen.getByText(/usd/i)).toBeInTheDocument();
+        expect(screen.getByRole('button', { name: /top up/i })).toBeInTheDocument();
+        expect(screen.getByRole('button', { name: /trade/i })).toBeInTheDocument();
+    });
+
+    it('should render the account card if the user has a Real MT5 Synthetic account', () => {
+        const type = {
+            type: 'synthetic',
+            category: 'real',
+            platform: 'mt5',
+        };
+        render(
+            <CFDAccountCard
+                {...props}
+                type={type}
+                descriptor={synthetic_descriptor}
+                title='Synthetic'
+                existing_accounts_data={[mt5_acc]}
+            />
+        );
+        expect(screen.getByText(/most popular/i)).toBeInTheDocument();
+        expect(screen.getByText(/IcMt5CfdPlatform/i)).toBeInTheDocument();
+        expect(screen.getAllByText(/synthetic/i)[0]).toBeInTheDocument();
+        expect(
+            screen.getByText(/trade cfds on our synthetic indices that simulate real-world market movement./i)
+        ).toBeInTheDocument();
+        expect(screen.getByText(/mfsa/i)).toBeInTheDocument();
```

Comment:
@thisyahlen-deriv unfortunately, mfsa will be soon changed to malta here which will lead to this test failing. Could you please test a non-eu account here or change the way you test this part to avoid this? Same for line 216 🙂

Reply:
On it 👍